<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskMergerGUI</class>
 <widget class="QWidget" name="MaskMergerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="m_LeftMask1Label">
       <property name="text">
        <string>left mask 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_LeftMask1ComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_LeftMask2Label">
       <property name="text">
        <string>left mask 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_LeftMask2ComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_RightMask1Label">
       <property name="text">
        <string>right mask 1:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_RightMask1ComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_RightMask2Label">
       <property name="text">
        <string>right mask 2:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_RightMask2ComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>151</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
